### ISSUE TITLE
Unable to compile on RHEL8

### ISSUE THREAD

**Issue by treydock** [stanford-rc/slurm-spank-lua#19]
```
+ gcc -g -o lua.o -fPIC -c lua.c
lua.c: In function 'SPANK_table_create':
lua.c:1009:23: error: assignment to expression with array type
         loadstr_table = "table.";
```

**Comment by basvandervlies** [comment]
We have the same problem on Debian 10/11 and had the same solution and pull request.

**Comment by kcgthb** [comment]
The fix has been merged and released in [v0.46](https://github.com/stanford-rc/slurm-spank-lua/releases/tag/v0.46)
